constraint/pkg/client/drivers/local/driver.go

```diff
@@ -72,6 +73,14 @@ type Driver struct {
 	clientCertWatcher *certwatcher.CertWatcher
 }
 
+// RegoEvaluationMeta has rego specific metadata from evaluation.
+type RegoEvaluationMeta struct {
+	// TemplateRunTime is the number of milliseconds it took to evaluate all constraints for a template.
+	TemplateRunTime float64 `json:"templateRunTime"`
```

Comment:
should we add the units to this field?

Reply:
K8s API conventions prefer integrating them into the field:

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units